src/pip/_internal/utils/logging.py

```diff
@@ -257,6 +257,10 @@ def setup_logging(verbosity: int, no_color: bool, user_log_file: Optional[str])
     # enabled for vendored libraries.
     vendored_log_level = "WARNING" if level in ["INFO", "ERROR"] else "DEBUG"
 
+    # Similar for vendored Kerberos, which is a bit trigger happy.
+    logging.addLevelName(logging.CRITICAL + 1, "SUPERCRITICAL")
+    kerberos_log_level = "SUPERCRITICAL" if level in ["INFO", "ERROR"] else "DEBUG"
```

Comment:
What if we simplify merge this into `CRITICAL`? This level in pip is already world-breaking level, and a level above it doesn’t make that much sense.

Reply:
We don't even vendor this code anymore, so we are not responsible anymore. Will remove changes in this file. 🙃